src/lib/DefaultRuleManager.ts

```diff
@@ -123,16 +125,21 @@ export class DefaultRuleManager implements RuleManager {
 						if (targetPatterns) {
 							// If dir, use globby { cwd: process.cwd() } option
 							const relativePaths = await globby(targetPatterns, {cwd: target});
-							ruleTargets.push({target, isDirectory: true, paths: relativePaths});
+							// Join the relative path to the files that were found
+							const joinedPaths = relativePaths.map(t => path.join(target, t));
+							// Resolve the relative paths to their absolute paths
+							const absolutePaths = joinedPaths.map(t => path.resolve(t));
+							ruleTargets.push({target, isDirectory: true, paths: absolutePaths});
 						} else {
 							// Without target patterns for the engine, just add the dir itself and hope for the best.
 							ruleTargets.push({target, isDirectory: true, paths: ["."]});
```

Comment:
I believe this is incorrect, but I couldn't figure out a way to actually hit the line to test it. Any ideas?